conversion_sr14_sr15/README.md

```diff
@@ -0,0 +1,74 @@
+# 1.0 Introduction
+The _conversion_sr14_sr15_ script converts Series 14 (SR14) data, used primarily in ABM2+, into Series 15 (SR15) format, which is used in ABM3. It adjusts SR14 land use and synthetic population files to match SR15 MGRA boundaries and uses an existing SR15 land use data to estimate SR15-equivalent land use columns. The exact processing procedures implemented are detailed in the following section. 
+
+# 2.0 Methodology
+## 2.1 Input Files
+Table 1 lists the input files required to run the conversion script and their purpose in the process. All files must be for the same scenario year. For clarity, SR15 refers to the converted SR14 file (output of this script), while ABM3 refers to the existing ABM3 land use input file (in SR15 format) that is used in the conversion process.
+
+**Table 1. Input Files**
+|File Name|Purpose|
+|---|---|
+|SR14 synthetic household file|Recoded to SR 15 MGRAs and fields renamed consistent with ABM3 field names. Also summarized for household totals to be used in converted MGRA data. |
+|SR14 synthetic person file| Fields renamed consistent with ABM3 field names. Summarized for person totals to be used in converted MGRA data.| 
+|SR14 land use file| Source of employment and income to be used in converted MGRA data file.|
+|ABM3 land use| Source of SR15 MGRAs, TAZs, acres, park space, parking costs, and a few other fields held constant in converted MGRA data.|
+|MGRA SR14 to SR15 crosswalk| Crosswalk used for conversion. Created via a separate process in which the centroid of each SR14 MGRA was geocoded to the polygon of SR15 MGRAs.|
+
+## 2.2 Data Processing
+### Households and Persons 
+The synthetic household and person files undergo minimal processing. For the household file, the MGRA column is updated using the SR14 to SR15 crosswalk and the column “hworkers” is renamed to “num_workers”. For the persons file, column ‘miltary’ is renamed to ‘military’, and columns ‘indcen’, ‘weeks’, ‘hours’, ‘rac1p’, ‘hisp’, and ‘version’ are removed, as they are not used in the modeling process. These files are used to calculate household and person totals in the converted SR15 land use file. 
+
+### Land Use
+The land use file undergoes extensive processing. There are three major procedures implemented depending on the columns: 
+1) The converted SR14 synthetic population is used to populate household and person total columns 
```

Comment:
Need clarification: the converted SR14 synthetic population is SR15 files, correct?

Reply:
Correct, the language has been updated to say: "The "SR15" synthetic population..."